### PR TITLE
Bm/feature/proptypes

### PR DESCRIPTION
**Pull request by AurumValian**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [x] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
This PR adds in Prop Types and removes linter errors.
### Files modified:
- [ ] index.html
- [x] App
- [x] Header
- [x] CardContainer
- [x] PokeCard
- [x] PokeForm
- [x] Login
- [x] FormContainer
- [ ] index.js
- [ ] Other files:

### CODE REVIEW

File: stretch-auth/src/FormContainer/FormContainer.js [hunk 1]
```diff
@@ -119,6 +114,15 @@ class FormContainer extends Component {
...
 }
+
+FormContainer.propTypes = {
+  currentUser: PropTypes.string,
+  allPokemon: PropTypes.array,
+  userPokeTeam: PropTypes.object,
+  savePokemonToUser: PropTypes.func,
+  updateUserPokeTeam: PropTypes.func
+}
+
```
Comment (jordy1611) [inline stretch-auth/src/FormContainer/FormContainer.js:125]: Nice! surprised currentUser is a string

**Review by jordy1611 — COMMENTED** [review]
Good implementation of prop types

**Review by H-Bo214 — APPROVED** [review]
Nice work Ben!

**Review by nware1066 — APPROVED** [review]
good to have those prop-types in the code. I have discussed your changes with Horacio, as well and he agrees to the merge, too

**Pull request merged**